pkg/virt-operator/install-strategy/create.go

```diff
@@ -662,16 +1465,143 @@ func SyncAll(kv *v1.KubeVirt,
 		if modified {
 			userBytes, err := json.Marshal(users)
 			if err != nil {
-				return false, err
+				return err
 			}
 
 			data := []byte(fmt.Sprintf(`{"users": %s}`, userBytes))
 			_, err = scc.SecurityContextConstraints().Patch(sccPriv.TargetSCC, types.StrategicMergePatchType, data)
 			if err != nil {
-				return false, fmt.Errorf("unable to patch scc: %v", err)
+				return fmt.Errorf("unable to patch scc: %v", err)
 			}
 		}
 	}
+	return nil
+}
+
+func SyncAll(kv *v1.KubeVirt,
+	prevStrategy *InstallStrategy,
+	targetStrategy *InstallStrategy,
+	stores util.Stores,
+	clientset kubecli.KubevirtClient,
+	expectations *util.Expectations) (bool, error) {
+
+	kvkey, err := controller.KeyFunc(kv)
+	if err != nil {
+		return false, err
+	}
+
+	gracePeriod := int64(0)
+	deleteOptions := &metav1.DeleteOptions{
```

Comment:
Only pods support `GracePeriodSeconds` and there we would not want to override the preconfigured grace period. Can we remove that?

Reply:
yeah, I think this can get removed.